cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/CaReconciler.java

```diff
@@ -381,7 +451,7 @@ Future<Void> rollingUpdateForNewCaKey() {
      *
      * @return  Current number of ZooKeeper replicas
      */
-    private Future<Integer> getZooKeeperReplicas() {
+    Future<Integer> getZooKeeperReplicas() {
```

Comment:
Is it used in tests? Should it be in that case
```suggestion
    /* test */ Future<Integer> getZooKeeperReplicas() {
```
as we normally do for other methods?

Reply:
Yep done.